docs/plugins/retry.md

```diff
@@ -0,0 +1,23 @@
+# Retry
+
+The retry plugin allows you to configure retry rules for your proxy. This enables you to be more resilient for any network or any other kind of failure.
+
+## Configuration
+
+The plain retry config:
+
+```json
+"retry": {
+    "enabled": true,
+    "config": {
+        "attempts": 3,
+        "backoff": "1s",
+        "predicate": "statusCode == 0 || statusCode >= 500"
+    }
+}
+```
+
+| Configuration | Description        |
+| attempts      | Number of attempts |
+| backoff       | Time that we should wait to retry. This can must be given in the format of `1s`, `5M`, `2H`. Defaults to `1s` |
```

Comment:
If it uses `time.Duration()` for parsing this field value we could give a link to a man - https://golang.org/pkg/time/#ParseDuration